openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.component.tsx

```diff
@@ -313,6 +315,13 @@ export const DataAssetsHeader = ({
             </Col>
             <Col span={24}>
               <div className="d-flex no-wrap">
+                <DomainLabel
+                  domain={(dataAsset as Table).domain}
```

Comment:
Instead add check like https://github.com/open-metadata/OpenMetadata/blob/768356009413abb3a80a680827dbc208e6b1f756/openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.component.tsx#L146

Reply:
It again type error because MetadataService doesn't contain domain